strabo/static/js/upload_ips.js

```diff
@@ -1,50 +1,124 @@
-var layer_message;
-var icon_message;
-var map_message;
-var title_message;
+var InputField = function(message_identifier,is_valid_cond){
```

Comment:
In terms of functionality, it is exactly the same. I just like the way it works more, the way that the id of the error and the condition are right next to each other in the code and the way the logic happens in a loop.